apps/src/makerlab/dropletConfig.js

```diff
@@ -0,0 +1,64 @@
+/* global dashboard */
+
+import api from '../applab/api';
+
+const COLOR_LIGHT_GREEN = '#D3E965';
+const COLOR_CYAN = '#4DD0E1';
+
+const playSongConfig = {
+  song: [
+    ["A", 500], [null, 50], ["A", 500], [null, 50], ["A", 500], [null, 50],
+    ["F", 350], [null, 50], ["C5", 150], [null, 50], ["A", 500], [null, 50],
+    ["F", 350], [null, 50], ["C5", 150], [null, 50], ["A", 650], [null, 50],
+    [null, 500], ["E5", 500], [null, 50], ["E5", 500], [null, 50], ["E5", 500],
+    [null, 50], ["F5", 350], [null, 50], ["C5", 150], [null, 50], ["G4", 500],
+    [null, 50], ["F", 350], [null, 50], ["C5", 150], [null, 50], ["A", 650],
+    [null, 50], [null, 500],
+  ],
+  tempo: 100000
+};
+
+const pixelType = '[Pixel]';
+const colorPixelVariables = ['pixels', 'pixel0', 'pixel1', 'pixel2', 'pixel3', 'pixel4', 'pixel5', 'pixel6', 'pixel7', 'pixel8', 'pixel9'];
+const simpleLedVariables = ['led'].concat(colorPixelVariables);
```

Comment:
this could also be `const simpleLedVariables = ['led', ...colorPixelVariables];`